### ISSUE TITLE
When running optimization job, it happen "Error: Re-submitting job"

### ISSUE THREAD

**Issue by NtuCheRoy** [ReactionMechanismGenerator/ARC#201]
I try to submit a opt job. It appear a `Re-submitting job` error. However, when I run sp job, it doesn't happen this error. Below is trace message.
```
ARC execution initiated on Thu Sep 19 06:53:09 2019

###############################################################
#                                                             #
#                 Automatic Rate Calculator                   #
#                            ARC                              #
#                                                             #
#   Version: 1.1.0                                            #
#                                                             #
###############################################################

The current git HEAD for ARC is:
    9fc5c7722322bc0a0b74c82652cc8401f8fa3fae
    Mon Sep 16 22:39:52 2019 -0400


Starting project hydrogen

Considering the following job types: ['opt']


Using the following ESS settings:
{'qchem': ['local']}



Warning: Not using a fine grid for geometry optimization jobs




Warning: Not running rotor scans. This might compromise finding the best conformer, as dihedral angles won't be corrected. Also, entropy won't be accurate.


Using b3lyp/6-31g* for refined conformer searches (after filtering via force fields)
Using b3lyp/6-31g* for TS guesses comparison of different methods
Using b3lyp/6-31g* for geometry optimizations
Using b3lyp/6-31g* for frequency calculations
Using b3lyp/6-31g* for single point calculations
Using b3lyp/6-31g* for rotor scans



Warning: Could not determine appropriate Model Chemistry to be used in Arkane for thermochemical parameter calculations.
Not using atom energy corrections and bond additivity corrections!


Using b3lyp/6-31g*//b3lyp/6-31g* as a model chemistry in Arkane
Warning: :root:No frequency scaling factor found for model chemistry b3lyp/6-31g*. Assuming a value of unity. This will affect the partition function and all quantities derived from it (thermo quantities and rate coefficients).
Could not determine the harmonic frequencies scaling factor for b3lyp/6-31g* from Arkane.
Not calculating it, assuming a frequencies scaling factor of 1.


Considering species: hydrogen
<Molecule "[H][H]">


Only one conformer is available for species hydrogen, using it as initial xyz
The only conformer for species hydrogen was found to be isomorphic with the 2D graph representation [H][H]

Running job opt_a72 for hydrogen
  Ending job opt_a72 for hydrogen (run time: 0:00:22)


Warning: Troubleshooting hydrogen job opt_a72 which failed with status "errored" with keywords [u'Unknown'] in qchem. The error "QChem job terminated for an unknown reason." was derived from the following line in the log file: "".
Error: Re-submitting job opt_a72 on local
Traceback (most recent call last):
  File "/global/homes/s/shcheng/ARC/ARC.py", line 76, in <module>
    main()
  File "/global/homes/s/shcheng/ARC/ARC.py", line 70, in main
    arc_object.execute()
  File "/global/u2/s/shcheng/ARC/arc/main.py", line 718, in execute
    dont_gen_confs=self.dont_gen_confs)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 391, in __init__
    self.schedule_jobs()
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 454, in schedule_jobs
    success = self.parse_opt_geo(label=label, job=job)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 1759, in parse_opt_geo
    self.troubleshoot_opt_jobs(label=label)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 2306, in troubleshoot_opt_jobs
    self.troubleshoot_ess(label=label, job=job, level_of_theory=self.opt_level)
  File "/global/u2/s/shcheng/ARC/arc/scheduler.py", line 2334, in troubleshoot_ess
    job.troubleshoot_server()
  File "/global/u2/s/shcheng/ARC/arc/job/job.py", line 1118, in troubleshoot_server
    server_nodes=self.server_nodes)
  File "/global/u2/s/shcheng/ARC/arc/job/trsh.py", line 708, in trsh_job_on_server
    ssh = SSHClient(server)
  File "/global/u2/s/shcheng/ARC/arc/job/ssh.py", line 49, in __init__
    self.address = servers[server]['address']
KeyError: u'address'
```

**Comment by alongd** [comment]
Please upload the output file of opt_a72 (I think you'll need to rename the extension to .txt for GitHub to accept it as an attachment to a comment)

**Comment by NtuCheRoy** [comment]
[output.txt](https://github.com/ReactionMechanismGenerator/ARC/files/3639242/output.txt)

**Comment by alongd** [comment]
This is indeed a bug, thanks for reporting! Should be solved via #207
